sklearn/metrics/pairwise.py

```diff
@@ -28,6 +28,7 @@
 from ..utils import Parallel
 from ..utils import delayed
 from ..utils import effective_n_jobs
+from ..neighbors import DistanceMetric
```

Comment:
Make this a local import. I don't think this is an acceptable dependency.

Reply:
done